numpy/lib/function_base.py

```diff
@@ -1557,28 +1557,55 @@ def sort_complex(a):
         return b
 
 
-def _trim_zeros(filt, trim=None):
+def _trim_zeros(
+    filt,
+    trim=None,
+    axis=None,
+    *,
+    atol=None,
+    rtol=None,
+    return_lengths=None
+):
     return (filt,)
 
 
 @array_function_dispatch(_trim_zeros)
-def trim_zeros(filt, trim='fb'):
-    """
-    Trim the leading and/or trailing zeros from a 1-D array or sequence.
+def trim_zeros(
+    filt,
+    trim='fb',
+    axis=-1,
```

Comment:
Not sure about the proper default value here. `None` (trim all dimensions), 0 or 1 (trim first/last dimension) are candidates that would preserve the old behavior.